src/lib.rs

```diff
@@ -108,12 +110,16 @@ impl TargetDep {
 
             // Move output
             std::fs::rename(out_path, &self.output_path).unwrap_or_else(|err| {
-                panic!(
-                    "Failed to move output {:?} to {:?}: {}",
+                println!(
```

Comment:
`stderr` is safer for logging; `stdout` can carry build metadata in cargo build scripts (see line 125 below)
```suggestion
                eprintln!(
```

Reply:
done. thank you.